src/java/test/config/findbugsExcludeFile.xml

```diff
@@ -90,6 +90,16 @@
      <Bug code="IS"/>
   </Match>
   <Match>
+    <Class name="org.apache.zookeeper.server.quorum.Leader"/>
+    <Method name="lead"/>
```

Comment:
@rakeshadr Initially I avoided this change because I did not want to dirty up the diff and we were doing the correct things to prevent a race condition with these variables. I should have checked 3.5 to see what we were doing there. I made the change to `QuorumPeer.tick` and `FastLeaderElection.logicalclock` but I did not make the change to `AuthFastLeaderElection.logicalclock` because I thought it would be best to avoid changing a deprecated class. Let me know if that's ok.